.eslintignore

```diff
@@ -1,5 +1,6 @@
 src/images/*
```

Comment:
Just to double check, this PR only adds in the components, but no new pages, correct? Just getting the repo to be ready for the demo in the future?

Reply:
Yes, just trying to keep the size of a PR reasonable.